### DISCUSSION TITLE
Is there still a need for win32 wheels?

### DISCUSSION THREAD

**Discussion by albertosottile** [py2exe/py2exe#157] [Polls]
Building and testing wheels for `win32` takes a lot of CI runtime (half of each run). Furthermore, a lot of packages used in the functional tests (e.g. `scipy`) stopped providing wheels for this target. This reduces the test coverage for these wheels. 

With this issue, I am trying to gauge if there is still demand for updated `win32` wheels for `py2exe`. Please vote in the issue and perhaps leave a comment, if you still need `win32` wheels, describing your use case. For example, 3.10+ `win32` are not tested at all due to the lack of available wheels for the test packages.

Note that the question concerns only new wheels, the existing wheels up to version 0.12.0.2 will still and always be available on PyPI.

**Comment by Athanasius** [comment]
Yes, EDMarketConnector is still 32-bit unless and until we get around to changing some nasty `ctypes` code to work on 64-bit Python.  There's been insufficient incentive for either of the main developers to do the necessary dive into the Windows API documentation to find the correct argument types to use for 64-bit (it's legacy code from the original developer many years ago).

**Reply by albertosottile** [reply]
Understood. Will you be happy to still have `win32` wheels, but largely untested? This is what is currently happening for 3.10 wheels, and will most likely happen also for 3.11.

**Reply by Athanasius** [reply]
Yes, that works.  It might also prod us to put in the work to finally become a 64-bit application, but no timeline on that.

I always do a local build and test with new python, py2exe or setuptools to catch any issues *before* they show up in GitHub workflow builds.

**Comment by LeonarddeR** [comment]
For NVDA, we are also using Win32 still. It would be very helpful if we could depend on wheels for that.
